### DISCUSSION TITLE
一点小功能添加及优化建议

### DISCUSSION THREAD

**Discussion by ctforcp** [liriliri/eruda#370] [Ideas]
### Console
- 增加：单个复制
- filter 直接输入出结果，空的时间显示所有，不再以弹出的方式交互

### Storage
- 调整：单独抽离出来 Storage
- 调整：key 显示不完的时候换行显示，而不再是省略号显示
- 增加：编辑功能，点击可以直接编辑key value
- 增加：一个可以增加的功能 点击后表格最后出现可填写的输入key value 参考编辑
- 调整：filter 直接输入出结果，空的时间显示所有，不再以弹出的方式交互
- 调整：点击查看的时候不跳转其它地方，直接在下方显示
- 调整了下布局

### Network
- 调整：filter 直接输入出结果，空的时间显示所有，不再以弹出的方式交互
- 调整：复制放在了左侧的图标一起了
- 调整：点击查看的时候不跳转其它地方，直接在下方显示
- 增加了其它的一些复制

![image](https://github.com/liriliri/eruda/assets/49541356/b303acd7-bf60-4753-b1a9-77d47e496d70)

**Comment by SoLongSally** [comment]
Translate.... make available for
